### ISSUE TITLE
Files chapter question

### ISSUE THREAD

**Issue by tstokke** [barbarer/py4e-int#161]
https://runestone.academy/ns/books/published/UNDCS160-Spring2024-Python/files/userNames.html

In 8.6 I see 2 activities, which I completed. At the bottom it tells me I have completed 3 of 4 activities. This seems confusing. I ended going to the Chapter activity page to see what I wanted me to do, which shows 3 activities. Where is the 4th?

It doesn't seem obvious that I need to run the provided Answer solution after I fix the 5 errors in the Question code.

**Comment by loua** [comment]
Runestone counts the **Answer** tab as an activity because it is configured as `activecode`. One solution is to change the **Answer** to `code`. This will render the solution code, but it won't be runnable. @barbarer is this an acceptable solution?

```
    .. tab:: Answer

        .. code:: python

            count = 0 # Start counting from zero
            fname = input('Enter the file name: ') # Close parentheses
            fhand = open(fname) # Open the correct file name

            for line in fhand:
                if line.startswith('Received:'):
                # Check at the beginning of the line, not the end
                    count = count + 1 # Correct indentation.
            fhand.close()
            print('There were', count, 'lines starting with "Received:" in the file', fname)
```

**Comment by barbarer** [comment]
Yes, it is fine to change it to just code rather than activecode.
